Fix documentation errors (#132)
CABLE-LSM/benchcab: docs/user_guide/index.md

:   directory that contains the log files, output files, and tasks for running CABLE. 

`runs/fluxsite/tasks`

> :   directory that contains task directories. CABLE runs are organised into tasks where a task consists of a branch (realisation), a meteorological forcing, and a science configuration. In the above directory structure, `<task>` uses the following naming convention:

Reviewer: ```suggestion
:   directory that contains task directories. A task consists of a CABLE run for a branch (realisation), a meteorological forcing, and a science configuration. In the above directory structure, `<task>` uses the following naming convention:
```
Is it a bit more direct this way?
